### PR TITLE
Add separated example for public instances

### LINKED ISSUE #69: Improve master authorized networks example
Currently, the example config is invalid (see #20) as it attempts to set up a `master authorized network` using the [internal CIDR block](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/examples/mysql-and-postgres/main.tf#L58).

We should improve this situation by:
1. Moving the regular (non-private) [example](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/examples/mysql-and-postgres/main.tf#L47) into its own separate example
2. Accepting a user input (variable) for master authorized network (with a narrow default).

### PR DESCRIPTION
**Pull request by paulpalamarchuk**
* Fix #69
 * Pin provider versions in test fixtures

### CODE REVIEW

File: examples/mysql-and-postgres-public/main.tf [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  region  = var.region
```
Comment (aaron-lane) [inline examples/mysql-and-postgres-public/main.tf]: This argument should not be necessary. The modules have `region` arguments.
```suggestion
```

File: examples/mysql-and-postgres-public/main.tf [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  version = "~> 2.13"
+}
+
+provider "google-beta" {
+  region  = var.region
```
Comment (aaron-lane) [inline examples/mysql-and-postgres-public/main.tf]:
```suggestion
```

File: examples/mysql-and-postgres-public/variables.tf [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  type    = string
+}
+
+variable "postgresql_version" {
+  default = "POSTGRES_9_6"
+  type    = string
+}
+
+variable "project_id" {
+  type = string
```
Comment (aaron-lane) [inline examples/mysql-and-postgres-public/variables.tf]: Variables should have descriptions.
```suggestion
  description = "The ID of the project in which resources will be provisioned."
  type        = string
```

File: examples/mysql-and-postgres-public/versions.tf [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_version = ">= 0.12"
```
Comment (aaron-lane) [inline examples/mysql-and-postgres-public/versions.tf]:
```suggestion
  required_version = "~> 0.12.6"
```

File: test/fixtures/mysql-ha/versions.tf [hunk 5]
```diff
@@ -15,5 +15,9 @@
  */
 
 terraform {
-  required_version = ">= 0.12"
+  required_version = "~> 0.12.0"
+  required_providers {
```
Comment (aaron-lane) [inline test/fixtures/mysql-ha/versions.tf]: Required providers should only be configured on functional modules and submodules, not examples or fixtures.

Comment (paulpalamarchuk) [reply test/fixtures/mysql-ha/versions.tf]: Moved to submodules

File: examples/mysql-and-postgres-public/main.tf (hunk 1 above)
Comment (paulpalamarchuk) [reply examples/mysql-and-postgres-public/main.tf]: Removed

File: examples/mysql-and-postgres-public/variables.tf (hunk 3 above)
Comment (paulpalamarchuk) [reply examples/mysql-and-postgres-public/variables.tf]: Description added

File: examples/mysql-and-postgres-public/versions.tf (hunk 4 above)
Comment (paulpalamarchuk) [reply examples/mysql-and-postgres-public/versions.tf]: Updated

**Pull request merged**